File: example/javalib/testing/5-test-stealing/foo/test/src/foo/RandomTestsI.java
package foo;

import org.junit.Test;

public class RandomTestsI extends RandomTestsUtils {
  @Test
  public void test1() throws Exception {
    testGreeting("Mars", 16);
  }

  @Test
  public void test2() throws Exception {
    testGreeting("North", 12);
  }

  @Test
  public void test3() throws Exception {
    testGreeting("Onyx", 14);
  }

  @Test
  public void test4() throws Exception {
    testGreeting("Phoenix", 15);
  }

  @Test
  public void test5() throws Exception {
    testGreeting("Quest", 13);
  }

  @Test
  public void test6() throws Exception {
    testGreeting("Rain", 11);
  }

  @Test
  public void test7() throws Exception {
    testGreeting("Sky", 17);
  }
}

File: example/kotlinlib/testing/5-test-stealing/foo/test/src/foo/RandomTestsI.kt
package foo

import org.junit.jupiter.api.Test

class RandomTestsI : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() {
        testGreeting("Mars", 16)
    }

    @Test
    @Throws(Exception::class)
    fun test2() {
        testGreeting("North", 12)
    }

    @Test
    @Throws(Exception::class)
    fun test3() {
        testGreeting("Onyx", 14)
    }

    @Test
    @Throws(Exception::class)
    fun test4() {
        testGreeting("Phoenix", 15)
    }

    @Test
    @Throws(Exception::class)
    fun test5() {
        testGreeting("Quest", 13)
    }

    @Test
    @Throws(Exception::class)
    fun test6() {
        testGreeting("Rain", 11)
    }

    @Test
    @Throws(Exception::class)
    fun test7() {
        testGreeting("Sky", 17)
    }
}
